config/routes.rb

```diff
@@ -21,6 +21,7 @@
   resources :users, only: %i[show new edit update create destroy] do
     resource :streak, only: %i[show]
     resources :cards, only: %i[index destroy]
+    resource :subscription, only: %i[destroy]
```

Comment:
@castrolem are we considering the case where users can update their subscription level? Or how that will work?

Reply:
We're not, right now you can only cancel to get the other plan.

Reply:
Since there can only be 1 plan active for a user at a time.

Reply:
ok if this is the case, then a user should be able to have many subscriptions but only one should be active at a time. Should we change the model to reflect this? This is also related to my previous comment https://github.com/debtcollective/fundraising/pull/107#discussion_r334593067

Reply:
Sounds good, let's make the changes on the model level.